src/types.ts

```diff
@@ -1,7 +1,20 @@
 export interface PlanXExportData {
   question: string;
-  responses: unknown;
-  metadata?: any;
+  responses: string | number | ResponseObject[] | Record<string, any> | any;
```

Comment:
So, I attempted to make the PlanXExportData type _more_ specific, but in doing so have obviously sprung many TS warnings and angered CI. 

I haven't made any adjustments to the TS config yet because I'd first like to better understand the previous use/motivation of the "unknown" type for "responses" - it's not one I've come across before and don't really get it! 

I _think_ responses should be able to be typed as: 
```
responses: string | number | ResponseObject[] | Record<any, any>
```
_without_ the trailing general "any" but alas that also threw many warnings/errors !

Would love some tips for sorting this out without immediately loosening tsconfigs if anyone's got em!

Reply:
Thanks for clarification of intentions/responsibilities about type signatures this module should care about & when versus what's for `planx-new` - this is exactly what I was in need of ! will rebase and update shortly :slightly_smiling_face: